src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { KeyValue, Col, Row } from '@folio/stripes-components';
+import { FormattedMessage } from 'react-intl';
+import BoolToCheckbox from '../../Utils/BoolToCheckbox';
+
+import css from './InterfaceView.css';
+
+const printKeyValue = (label, val, columnCount) => {
```

Comment:
it looks like new code. If it is - please don't create such functions, they makes no sense. And even if that is copy-paste - it's marked as green, so you could refactor this to something meaningful